pom.xml

```
@@ -445,4 +447,33 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>jdk9plus</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                    <surefireArgLine>--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/sun.reflect.annotation=ALL-UNNAMED -Dfile.encoding=UTF-8</surefireArgLine>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdk8v0</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <surefireArgLine> -Dfile.encoding=UTF-8</surefireArgLine>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdk8v1</id>
+            <activation>
+                <jdk>8</jdk>
+            </activation>
+            <properties>
+                <surefireArgLine> -Dfile.encoding=UTF-8</surefireArgLine>
+            </properties>
+        </profile>
```

IIRC `1.8` is the formal reference. Any source urges us to write the section for `<jdk>8</jdk>`?

```suggestion
        <profile>
            <id>jdk8</id>
            <activation>
                <jdk>1.8</jdk>
            </activation>
            <properties>
                <surefireArgLine> -Dfile.encoding=UTF-8</surefireArgLine>
            </properties>
        </profile>
```